jobs/base_model_validator_test.py

```diff
@@ -169,3 +173,51 @@ def test_validate_model_id(self):
                 beam_testing_util.equal_to([
                     errors.ModelInvalidIdError(invalid_id_model)
                 ]))
+
+
+class ValidatePostCommitIsPrivateTests(BaseModelValidatorTests):
+    def test_validate_post_commit_is_private_when_status_is_public(self):
+        with pipeline.TestPipeline(runner=direct_runner.DirectRunner()) as p:
+            invalid_commit_status = MockCommitLogEntryModel(
+                id='123',
+                created_on=self.year_ago,
+                last_updated=self.now,
+                commit_type='invalid-type',
+                user_id='',
+                post_commit_status='public',
+                post_commit_is_private=True,
+                commit_cmds=[])
+            pcoll = p | beam.Create([invalid_commit_status])
+
+            output = (pcoll | beam.ParDo(
+                base_model_validator.ValidatePostCommitIsPrivate()))
```

Comment:
Our convention for multiline pipes is to start the `|` on a new line:

```suggestion
            output = (
                pcoll
                | beam.ParDo(base_model_validator.ValidatePostCommitIsPrivate()))
```

Please fix here and elsewhere.

Reply:
Done